database/postgres/pg-schema.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE IF NOT EXISTS zips(
+  zip_code INTEGER NOT NULL,
+  property_tax_rate DECIMAL(5,3),
+  CONSTRAINT zip_code PRIMARY KEY (zip_code)
+);
+
+-- property id is not set to serial/auto-increment to preserve behavior of endpoints from 1 to 1,000,000
+CREATE TABLE IF NOT EXISTS properties(
+  property_id INTEGER NOT NULL,
```

Comment:
should this be SERIAL as well?

Reply:
For now I am having the for loop create the index so I can control it to be between 1 and 1,000,000 and know what endpoint websites to go to vs. if postgres assigns the index and starts at some other number. But for the final product, may consider switching it over to having postgres assign it. Thanks!